content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -19,10 +19,18 @@ Database views allow you to name and store queries. In relational databases, vie
 
 The `views` preview feature allows you to represent views in your Prisma schema with the `view` keyword. To use views in Prisma, follow these steps:
 
-- [Enable the `views` preview feature](#enable-the-views-preview-feature)
-- [Create a view in the underlying database](#create-a-view-in-the-underlying-database), either directly or as a [manual addition to a Prisma Migrate migration file](#use-views-with-prisma-migrate-and-db-push), or use an existing view
-- [Represent the view in your Prisma schema](#add-views-to-your-prisma-schema)
-- [Query the view in Prisma Client](#query-views-in-prisma-client)
+- [Enable the views preview feature](#enable-the-views-preview-feature)
+- [Create a view in the underlying database](#create-a-view-in-the-underlying-database)
+- [Use views with Prisma Migrate and db push](#use-views-with-prisma-migrate-and-db-push)
+- [Add views to your Prisma schema](#add-views-to-your-prisma-schema)
+  - [Write by hand](#write-by-hand)
+    - [Define a `view` block](#define-a-view-block)
+    - [Define fields](#define-fields)
+  - [Use introspection](#use-introspection)
+  - [Limitations](#limitations)
+    - [Unique Identifier](#unique-identifier)
+    - [Introspection](#introspection)
+- [Query views in Prisma Client](#query-views-in-prisma-client)
```

Comment:
Why the switch from an editorialized list of the concrete steps to a (auto-generated?) table of contents of the whole page?

Reply:
This was my bad, I was using a new Markdown extension in VSCode and apparently it generates a TOC on save...

Reply:
Haha ok. Interesting that it figure out where to put it. I remember having to use weird comments to make something like this work in the past.